controlplane/eks/api/v1alpha4/conversion.go

```diff
@@ -23,21 +23,41 @@ import (
 	"sigs.k8s.io/cluster-api-provider-aws/controlplane/eks/api/v1beta1"
 	clusterv1alpha4 "sigs.k8s.io/cluster-api/api/v1alpha4"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	utilconversion "sigs.k8s.io/cluster-api/util/conversion"
 	"sigs.k8s.io/controller-runtime/pkg/conversion"
 )
 
 // ConvertTo converts the v1alpha4 AWSManagedControlPlane receiver to a v1beta1 AWSManagedControlPlane.
 func (r *AWSManagedControlPlane) ConvertTo(dstRaw conversion.Hub) error {
 	dst := dstRaw.(*v1beta1.AWSManagedControlPlane)
 
-	return Convert_v1alpha4_AWSManagedControlPlane_To_v1beta1_AWSManagedControlPlane(r, dst, nil)
+	if err := Convert_v1alpha4_AWSManagedControlPlane_To_v1beta1_AWSManagedControlPlane(r, dst, nil); err != nil {
```

Comment:
@richardcase This is what's needed for the newly added fields for ipv6 too, right?

Reply:
Yes exactly :+1: 

Sorry i didn't get around to sending you the example